src/core/common/logging.cpp

```diff
@@ -148,6 +148,42 @@ void otDump(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aId, const
     otLogDump("%s", buf);
 }
 
+#ifdef OPENTHREAD_CONFIG_LOG_PREPEND_LEVEL
+const char *otLogLevelToString(otLogLevel aLevel)
+{
+    const char *retval;
+
+    switch (aLevel)
+    {
+    case kLogLevelNone:
+        retval = "-NONE-";
+        break;
+
+    case kLogLevelCrit:
```

Comment:
Suggestion: We can make the log-level 4 char fixed (which they are) and remove "-" from begging and end...What do you think?